core/ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/ReferenceDevices.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2022 The Android Open Source Project
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package com.google.samples.apps.nowinandroid.core.ui
+
+import androidx.compose.ui.tooling.preview.Preview
+
+import androidx.annotation.StringDef
+
+/**
+ * List with the pre-defined devices available to be used in the preview.
+ */
+object Devices {
+
+    // Reference devices
+    const val PHONE = "spec:shape=Normal,width=411,height=891,unit=dp,dpi=420"
+    const val FOLDABLE = "spec:shape=Normal,width=673,height=841,unit=dp,dpi=420"
```

Comment:
Are these different from the existing definitions provided by Compose? For example, https://developer.android.com/reference/kotlin/androidx/compose/ui/tooling/preview/Devices#FOLDABLE()

Reply:
added a todo to remove once this is fixed: https://issuetracker.google.com/issues/254528382